### ISSUE TITLE
[Regression] Tales of Xillia (and 2) crash during first screen when input is detected

### ISSUE THREAD

**Issue by TheSilverDen** [RPCS3/rpcs3#12465]
### Quick summary

Tales of Xillia and Tales of Xillia 2 both crash in the first screen (Bandai Namco logo) if there is any controller input (to skip it, etc.). They work if you do not have any controller inputs in this screen and let it pass by itself. 

### Details

Tested with the EU Version of both games

### Build with regression

v0.0.23-13834-fcd297ff

### Attach two log files

[RPCS3-13833WORKING.log.gz](https://github.com/RPCS3/rpcs3/files/9266435/RPCS3-13833WORKING.log.gz)
[RPCS3-13834-BROKEN.log.gz](https://github.com/RPCS3/rpcs3/files/9266437/RPCS3-13834-BROKEN.log.gz)

**Comment by Megamouse** [comment]
I don't see any crash in the log

**Comment by TheSilverDen** [comment]
You are right, it is actually infinitely loading. The game keeps playing but nothing happens and it is stuck at a grey screen. I don't know if it helps but I have attached another log with an up-to-date build. Can provide further information if needed, of course.

[RPCS3-14007.log.gz](https://github.com/RPCS3/rpcs3/files/9268496/RPCS3-14007.log.gz)

**Comment by Megamouse** [comment]
Please try with #12467

**Comment by Megamouse** [comment]
This seems to be neither connected to cellPad nor cellVdec (as was my second assumption, but LLE doesn't help).
So something else in the savestate PR caused this.
It only happens if you skip the intro.
I'll try to make a diff between two runs to see where there might be a difference.

**Comment by Megamouse** [comment]
It also softlocks in the old build. Albeit probably not as often.

**Comment by elad335** [comment]
Is this affected by #13917 ?

**Comment by Tzerinas** [comment]
No, I'm getting the issue on latest master. As soon as I try to skip rpcs3 just shows a white screen / continues to load indefinitely.

**Comment by darrena092** [comment]
Same issue here - attempting to skip causes it to hang.

**Comment by iriyap** [comment]
Same here on the latest build, not a big deal but would love to see it fixed.

**Comment by 13xforever** [comment]
Still an issue on v0.0.37-17994-d15b7a99
I can also add that you can skip on the second logo screen just fine, it only breaks on the first screen.

[RPCS3_xillia_skip1_no_boot.log.gz](https://github.com/user-attachments/files/20637081/RPCS3_xillia_skip1_no_boot.log.gz)
[RPCS3_xillia_skip2_boot.log.gz](https://github.com/user-attachments/files/20637082/RPCS3_xillia_skip2_boot.log.gz)

**Comment by digant73** [comment]
reported as regressed by #10478
